project/First-Project/src/main/java/first/project/controller/AuthenticationController.java

```diff
@@ -0,0 +1,63 @@
+package first.project.controller;
+
+import first.project.acessingdatamysql.User;
+import first.project.acessingdatamysql.UserRepository;
+import first.project.service.AuthenticationService;
+import first.project.service.AuthenticationServiceImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+
+import java.util.List;
+
+@Controller
+public class AuthenticationController
+{
+    @Autowired
+    private AuthenticationService authenticationService;
+
+    @GetMapping("")
+    public String Home(Model model)
+    {
+        model.addAttribute("user", new User());
```

Comment:
@HristijanSlavkoski ping

Reply:
@aleksandar-bujaroski-armedia Dejan told me if we enter only localhost:8080, it should redirect us to register page because we won't type /register as users. Actually home page is link to register page.